source/Plx.js

```diff
@@ -698,10 +698,7 @@ function getNewState(scrollPosition, props, state, element) {
   newStyle.OFilter = newStyle.filter;
   newStyle.msFilter = newStyle.filter;
 
-  // "Stupid" check if style should be updated
-  if (JSON.stringify(plxStyle) !== JSON.stringify(newStyle)) {
-    newState.plxStyle = newStyle;
-  }
+  newState.plxStyle = newStyle
```

Comment:
This probably gives the biggest performance boost (`stringify` is expensive), but does it affect performance with with more state updates?

Reply:
> This probably gives the biggest performance boost (`stringify` is expensive), but does it affect performance with more state updates?

Ah, wrong guess. The mathmatical decimal places change gives most, I remember. But this is also expensive. I thought it did not call `setState` method and is merely an object modification, and in JS the assignment is passing a reference here. If it does have side effects, I may need further confirmation.

Reply:
I actually update state here:
https://github.com/Stanko/react-plx/blob/master/source/Plx.js#L713-L715

And that depends if `newState` has any keys.

But let's go with your change.